docs/troubleshooting/semgrep-app.md

```diff
@@ -1,57 +1,188 @@
 ---
 slug: semgrep-app
 description: "Not seeing what you expect in Semgrep AppSec Platform? Follow these troubleshooting steps or find out how to get one-on-one help."
-title: Troubleshooting
+title: Troubleshooting CI
 hide_title: true
 tags:
+    - Deployment
+    - Troubleshooting
     - Semgrep AppSec Platform
 ---
 
-# Troubleshooting Semgrep AppSec Platform
+import RetrieveGhaLogs from "/src/components/procedure/_retrieve-gha-logs.mdx"
+
+# Troubleshooting CI scans
+
+This document outlines troubleshooting steps for issues related to **Semgrep scans** in a CI environment. Refer to the following sections if you're seeing results reported on files that have not changed since the last scan, frequent timeouts, or other issues.
+
+For issues on **deployment or CI configuration**, such as adding repositories, see the knowledge base articles in [<i class="fa-regular fa-file-lines"></i> Semgrep in CI](/kb/semgrep-ci).
+
+## Reproducing the issue locally
+
+To aid in debugging, you can reproduce some aspects of your Semgrep CI job locally. This enables you to inspect the logs and behavior through your terminal rather than in your CI provider's interface. Perform the following steps:
+
+1. Run the following command in your terminal:
+    ```
+    semgrep login
+    ```
+1. After logging in, return to the CLI and enter the following: <pre class="language-bash"><code>SEMGREP_REPO_NAME=<span className="placeholder">your-organization</span>/<span className="placeholder">repository-name</span> semgrep ci</code></pre>
+  For example, given a GitHub repository is `vulncorp/juice-shop`, the full command would be:
+  ```
+  SEMGREP_REPO_NAME=vulncorp/juice-shop semgrep ci
+  ```
```

Comment:
I think the above code block needs additional indentation so it lines up properly